test/unit/puppetlabs/services/master/master_core_test.clj

```diff
@@ -28,7 +28,9 @@
   [task-name]
   (let [[module _] (split task-name #"::")]
     {:module {:name module}
-     :name task-name}))
+     :name task-name
+     :private false
```

Comment:
I think these mocks are wrong. I believe `private` and `description` should be nested under `metadata`.  https://github.com/puppetlabs/puppet/pull/8921/files